pycodestyle.py

```diff
@@ -124,7 +124,7 @@
 STARTSWITH_DEF_REGEX = re.compile(r'^(async\s+def|def)')
 STARTSWITH_TOP_LEVEL_REGEX = re.compile(r'^(async\s+def\s+|def\s+|class\s+|@)')
 STARTSWITH_INDENT_STATEMENT_REGEX = re.compile(
-    r'^\s*({0})'.format('|'.join(s.replace(' ', '\s+') for s in (
+    r'^\s*({0})\s'.format('|'.join(s.replace(' ', '\s+') for s in (
```

Comment:
Haha, that's a good point! Though might as well not break this rule, and let that abomination be caught by E2xx rule.